mvn clean install fails

Hi,

I was trying to launch your project but when I launch `mvn clean install`, here's the error I have:

```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.deeplearning4j:scalphagozero:[unknown-version]: Could not find artifact org.deeplearning4j:deeplearning4j:pom:1.0.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 7, column 13
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.deeplearning4j:scalphagozero:[unknown-version] (/Users/jules/colisweb/workspace/ScalphaGoZero/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.deeplearning4j:scalphagozero:[unknown-version]: Could not find artifact org.deeplearning4j:deeplearning4j:pom:1.0.0-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 7, column 13 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```

I'm a maven noob. 😕

How can I fix that ?

Thanks :)

--- a/pom.xml
+++ b/pom.xml
@@ -4,16 +4,11 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
-    <parent>
-        <groupId>org.deeplearning4j</groupId>
-        <artifactId>deeplearning4j</artifactId>
-        <version>1.0.0-SNAPSHOT</version>
-    </parent>
-
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.deeplearning4j</groupId>
     <artifactId>scalphagozero</artifactId>
+    <version>1.0.0</version>
 
     <name>ScalphaGoZero</name>
     <description>An independent implementation of DeepMind's AlphaGoZero in Scala, using Deeplearning4J (DL4J)
@@ -35,6 +30,11 @@
     </developers>
 
     <properties>
+        <scalafmt.version>1.3.0</scalafmt.version>
+        <nd4j.version>0.9.1</nd4j.version>
+        <scalatest.version>3.0.5</scalatest.version>
+        <logback.version>1.2.3</logback.version>
+        <dl4j.version>1.0.0-beta2</dl4j.version>
         <scala.version>2.11.12</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
     </properties>

--- a/pom.xml
+++ b/pom.xml
@@ -31,10 +31,10 @@
 
     <properties>
         <scalafmt.version>1.3.0</scalafmt.version>
-        <nd4j.version>0.9.1</nd4j.version>
+        <nd4j.version>1.0.0-beta2</nd4j.version>
+        <dl4j.version>1.0.0-beta2</dl4j.version>
         <scalatest.version>3.0.5</scalatest.version>
         <logback.version>1.2.3</logback.version>
-        <dl4j.version>1.0.0-beta2</dl4j.version>
         <scala.version>2.11.12</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
     </properties>

